File: src/tests/filter_bilateral_2df_test/filter_bilateral_2df_test.cpp
#include <GL/glew.h>
#include <piccante.hpp>

int main()
{
    pic::Image imgIn("pacman.png",pic::LDR_type::LT_NOR);

    pic::Image imgOut = pic::FilterBilateral2DF::execute(&imgIn, &imgOut, 1.5, 100.6);

    imgOut.Write("filtered.png");
}

File: src/tests/filter_bilateral_2df_test/filter_bilateral_2df_test.py
import pyccante as py

imgIn = py.Image("pacman.png", py.LDR_type.LT_NOR)
imgOut = py.Image()
imgOut = py.FilterBilateral2DF.execute(imgIn, imgOut, 1.5, 100.6)
imgOut.Write("filtered.png")